euphony/src/main/java/co/euphony/ui/EuphonyRxPanel.kt

```diff
@@ -0,0 +1,119 @@
+package co.euphony.ui
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.euphony.common.Constants
+import co.euphony.rx.EuRxManager
+import co.euphony.ui.theme.*
+import co.euphony.ui.viewmodel.EuphonyRxPanelViewModel
+
+@Composable
+fun EuphonyRxPanel() {
+    val viewModel = EuphonyRxPanelViewModel(EuRxManager())
+    EuphonyRxPanelImpl(viewModel = viewModel)
+}
+
+@Composable
+fun EuphonyRxPanelImpl(
+    viewModel: EuphonyRxPanelViewModel
+) {
```

Comment:
I want user to use only `EuphonyRxPanel`, not `EuphonyRxPanelmpl`, so how about defining as `internal fun`?